src/LRS.js

```diff
@@ -456,12 +449,34 @@ TinCan client library
                     statementId: stmtId
                 }
             };
+            if (cfg.params.attachments) {
+                requestCfg.params.attachments = true;
+            }
             if (typeof cfg.callback !== "undefined") {
                 callbackWrapper = function (err, xhr) {
-                    var result = xhr;
+                    var result = xhr.responseText,
```

Comment:
Changing the initialization of `result` here breaks the public interface contract in the case of errors. (Which I honestly thought would have been caught by the unit tests :-(). See `cfg.callback(err, result)` below.